packages/react-native-bpk-theming/src/types.js

```diff
@@ -19,22 +19,19 @@
 /* @flow */
 
 export type Theme = {
+  buttonBorderRadius?: Number,
```

Comment:
Should this have changed from `number` to `Number`?

Reply:
Good catch, it's my bloody editor that keeps suggesting it and sneaks it in without me meaning to.

Reply:
Fixed now